logging/logging.go

```diff
@@ -884,15 +884,15 @@ var validXCloudTraceContext = regexp.MustCompile(
 		`(?:;o=(\d))?`)
 
 func deconstructXCloudTraceContext(s string) (traceID, spanID string, traceSampled bool) {
-	// As per the format described at https://cloud.google.com/trace/docs/setup#force-trace
-	//    "X-Cloud-Trace-Context: TRACE_ID/SPAN_ID;o=TRACE_TRUE"
+	// As per the format described at https://cloud.google.com/trace/docs/trace-context#legacy-http-header
+	//    "X-Cloud-Trace-Context: TRACE_ID[/SPAN_ID][;o=TRACE_TRUE]"
 	// for example:
-	//    "X-Cloud-Trace-Context: 105445aa7843bc8bf206b120001000/1;o=1"
+	//    "X-Cloud-Trace-Context: 105445aa7843bc8bf206b12000100000/1;o=1"
 	//
 	// We expect:
```

Comment:
It might be easier to understand if you phrase the comment as:

```
  // Parameters:
  //   s - ...
  //
  // Returns:
  //     ...
```

With respect to "We expect", it is unclear whether this is referring to the input or the output.

If it is intended to describe the input, then shouldn't the documentation for `spanID` say it is a decimal string?